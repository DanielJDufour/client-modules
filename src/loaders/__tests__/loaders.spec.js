const loaders = {
  babel: require('../babel'),
  files: require('../files'),
  styles: require('../styles')
}

describe('Loaders', () => {
  it('matches snapshots', () => {
    expect(loaders).toMatchSnapshot();
  })
});
