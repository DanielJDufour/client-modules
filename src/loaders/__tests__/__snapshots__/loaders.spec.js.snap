exports[`Loaders matches snapshots 1`] = `
Object {
  "babel": Object {
    "default": Object {
      "loader": "babel-loader",
      "options": Object {
        "cacheDirectory": true,
      },
      "test": /\\.js?$/,
    },
  },
  "files": Object {
    "default": Object {
      "loader": "file-loader",
      "test": /\\.jpe?g$|\\.gif$|\\.png$|\\.svg$|\\.woff$|\\.ttf$|\\.eot|\\.wav$|\\.mp3$/,
    },
  },
  "styles": Object {
    "css": Object {
      "default": Object {
        "test": /\\.css?$/,
        "use": Array [
          Object {
            "loader": "style-loader",
          },
          Object {
            "loader": "css-loader",
            "options": Object {
              "importLoaders": 1,
              "localIdentName": "[name]__[local]___[hash:base64:5]",
              "sourceMap": true,
            },
          },
          Object {
            "loader": "postcss-loader",
            "options": Object {
              "plugins": [Function],
            },
          },
        ],
      },
      "extracted": Object {
        "test": /\\.css?$/,
        "use": Array [
          Object {
            "loader": "/Users/jake/Sites/Codecademy/npm/webpack-config/node_modules/extract-text-webpack-plugin/loader.js",
            "options": Object {
              "omit": 1,
              "remove": true,
            },
          },
          Object {
            "loader": "style-loader",
          },
          Object {
            "loader": "css-loader",
            "options": Object {
              "importLoaders": 1,
              "localIdentName": "[name]__[local]___[hash:base64:5]",
              "sourceMap": true,
            },
          },
          Object {
            "loader": "postcss-loader",
            "options": Object {
              "plugins": [Function],
            },
          },
        ],
      },
      "server": Object {
        "test": /\\.css?$/,
        "use": Array [
          Object {
            "loader": "css-loader/locals",
            "options": Object {
              "importLoaders": 1,
              "localIdentName": "[name]__[local]___[hash:base64:5]",
              "sourceMap": true,
            },
          },
        ],
      },
    },
    "scss": Object {
      "default": Object {
        "test": /\\.scss?$/,
        "use": Array [
          Object {
            "loader": "style-loader",
          },
          Object {
            "loader": "css-loader",
            "options": Object {
              "importLoaders": 1,
              "localIdentName": "[name]__[local]___[hash:base64:5]",
              "modules": true,
              "sourceMap": true,
            },
          },
          Object {
            "loader": "postcss-loader",
            "options": Object {
              "plugins": [Function],
            },
          },
          Object {
            "loader": "sass-loader",
            "options": Object {
              "sourceMap": true,
            },
          },
        ],
      },
      "extracted": Object {
        "test": /\\.scss?$/,
        "use": Array [
          Object {
            "loader": "/Users/jake/Sites/Codecademy/npm/webpack-config/node_modules/extract-text-webpack-plugin/loader.js",
            "options": Object {
              "omit": 1,
              "remove": true,
            },
          },
          Object {
            "loader": "style-loader",
          },
          Object {
            "loader": "css-loader",
            "options": Object {
              "importLoaders": 1,
              "localIdentName": "[name]__[local]___[hash:base64:5]",
              "modules": true,
              "sourceMap": true,
            },
          },
          Object {
            "loader": "postcss-loader",
            "options": Object {
              "plugins": [Function],
            },
          },
          Object {
            "loader": "sass-loader",
            "options": Object {
              "sourceMap": true,
            },
          },
        ],
      },
      "server": Object {
        "test": /\\.scss?$/,
        "use": Array [
          Object {
            "loader": "css-loader/locals",
            "options": Object {
              "importLoaders": 1,
              "localIdentName": "[name]__[local]___[hash:base64:5]",
              "modules": true,
              "sourceMap": true,
            },
          },
          Object {
            "loader": "postcss-loader",
            "options": Object {
              "plugins": [Function],
            },
          },
          Object {
            "loader": "sass-loader",
            "options": Object {
              "sourceMap": true,
            },
          },
        ],
      },
    },
  },
}
`;
